test/functional/test-ampdoc.js

```diff
@@ -166,7 +166,8 @@ describe('AmpDocService', () => {
       }).to.throw(/The shadow root already contains ampdoc/);
     });
 
-    it('should navigate via host', () => {
+    // TODO(dvoytenko): Make this test work on Safari. See #11827.
```

Comment:
nit: `// TODO(dvoytenko, #11827)` (Dima has a tool that extracts these if the TODO(username, #issue) format is followed.

Reply:
Fixed.